packages/base/Autocomplete/src/Autocomplete/Autocomplete.tsx

```diff
@@ -17,28 +17,33 @@ import type { PopperOptions } from 'popper.js'
 import { Input } from '@toptal/picasso-input'
 import { Container } from '@toptal/picasso-container'
 import { Loader } from '@toptal/picasso-loader'
-import { SelectOptions } from '@toptal/picasso-select'
 import { Popper } from '@toptal/picasso-popper'
 import { InputAdornment } from '@toptal/picasso-input-adornment'
 import {
   unsafeErrorLog,
   usePropDeprecationWarning,
 } from '@toptal/picasso-utils'
 import type { InputProps } from '@toptal/picasso-input'
-import { MenuItem } from '@toptal/picasso-menu'
 import type { BaseInputProps, Status } from '@toptal/picasso-outlined-input'
 import { useFieldsLayoutContext } from '@toptal/picasso-form'
 import { twMerge } from '@toptal/picasso-tailwind-merge'
 
-import PoweredByGoogle from './PoweredByGoogle'
-import NoOptionsMenuItem from './NoOptionsMenuItem'
-import OtherOptionMenuItem from './OtherOptionMenuItem'
 import type { Item, ChangedOptions } from './types'
 import { useAutocomplete, EMPTY_INPUT_VALUE } from './use-autocomplete'
 import { rootClassByWidth } from './styles'
+import type { OptionsMenuProps } from './OptionsMenu'
+import OptionsMenu from './OptionsMenu'
 
 export interface Props
   extends BaseProps,
+    Omit<
+      OptionsMenuProps,
+      | 'getDisplayValue'
+      | 'highlightedIndex'
+      | 'shouldShowOtherOption'
+      | 'getItemProps'
+      | 'getOtherItemProps'
+    >,
```

Comment:
To make this stricter and future-proof. Instead of omitting other props I would only pick the ones we already support, so in the case new props are added to `OptionsMenu` we don't have to add them here:
```suggestion
    Pick<
      OptionsMenuProps,
      | 'otherOptionText'
      | 'renderOtherOption'
      | 'noOptionsText'
      | 'renderOption'
      | 'poweredByGoogle'
      | 'getDisplayValue'
    >,
```

Reply:
@augustobmoura Agree, addressed, thank you!